Data-Structures/src/main/java/linkedlist/LinkedList.java

```diff
@@ -0,0 +1,38 @@
+package linkedlist;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LinkedList {
+  public Node head;
+  public Node current;
+
+  public void insert(String value){
+    if(head == null){
+      head = new Node(value);
+      return;
+    }
+    head.next = new Node(value);
```

Comment:
This line of code will always insert a new node as the head's next. This means that the previous node the head was pointing to will be lost (along with any nodes that are connected to it). This method should instead put the new node as the linked list's head, with the previous head becoming `head.next`